src/strategies/erc20-balance-of/index.ts

```diff
@@ -3,7 +3,7 @@ import { formatUnits } from '@ethersproject/units';
 import { Multicaller } from '../../utils';
 
 export const author = 'bonustrack';
-export const version = '0.1.1';
+export const version = '0.1.2';
```

Comment:
```suggestion
export const version = '0.1.1';
```

Reply:
sorry for the mistake, changed the wrong file.
already pushed the fix